e2e/commands/checkAccessibility.js

```diff
@@ -1,7 +1,7 @@
 const { rootSelector } = require('../config')
 
 exports.command = function checkAccessibility() {
-  this.initAccessibility().assert.accessibility(rootSelector, {
+  this.initAccessibility().verify.accessibility(rootSelector, {
```

Comment:
What's going on here?

Reply:
When you use assert, the script will exit on failure. Verify does the exact same thing as assert except that failures don't exit the script.

Reply:
🆗